tests/unittests/test_request_timeout.py

```diff
@@ -0,0 +1,125 @@
+import unittest
+from unittest import mock
+from unittest.case import TestCase
```

Comment:
I don't think this import is necessary, since TestCase is reference as unittest.TestCase

Reply:
removed the import